openssl-sys/build.rs

```diff
@@ -44,7 +44,11 @@ fn main() {
     let libs = if (version.contains("0x10001") ||
                    version.contains("0x10002")) &&
                   target.contains("windows") {
-        ["ssleay32", "libeay32"]
+        if target.contains("windows-gnu") {
+            ["ssl", "crypto"]
```

Comment:
The original code indicates that the filenames are `ssleay32` & `libeay32` for OpenSSL 1.01 and 1.02 under Windows. I'm not sure whether this holds true with OpenSSL 1.1.0. If it is, I could edit this PR; if it's not, i.e. they're renamed to `ssl` and `crypto` again from 1.1.0, it will fall back to the last `else` and use the correct ones.